src/game/boe.newgraph.cpp

```diff
@@ -155,7 +155,7 @@ void apply_light_mask(bool onWindow) {
 	rectangle temp = {0,0,108,84},paint_rect,base_rect = {0,0,36,28};
 	rectangle big_to = {13,13,337,265};
 	bool same_mask = true;
-	if(!get_bool_pref("DrawTerrainFrills", true) || fog_lifted)
+	if(fog_lifted)
```

Comment:
This one has the same weird situation as the above one, where a strict reading would suggest deleting the entire function, but that can't be right.